lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb

```diff
@@ -45,29 +40,19 @@ def self.localize_project
           Fastlane::Helper::GitHelper.commit(message: 'Update strings for localization', files: strings_files, push: true) || UI.message('No new strings, skipping commit')
         end
 
-        # Call the `Scripts/update-translations.rb` then the `fastlane/download_metadata` Scripts from the host project folder
-        #
-        # @env PROJECT_ROOT_FOLDER The path to the git root of the project
-        # @env PROJECT_NAME The name of the directory containing the project code (especially containing the `build.gradle` file)
-        #
-        # @todo Migrate the scripts, currently in each host repo and called by this method, to be helpers and actions
-        #       in the release-toolkit instead, and move this code away from `ios_git_helper`.
-        #
-        def self.update_metadata
-          Action.sh("cd #{get_from_env!(key: 'PROJECT_ROOT_FOLDER')} && ./Scripts/update-translations.rb")
-
-          Fastlane::Helper::GitHelper.commit(message: 'Update translations', files: strings_files, push: false)
-
-          Action.sh('cd fastlane && ./download_metadata.swift')
+        def self.strings_files
+          project_root = env_project_root()
+          project_name = env_project_name()
```

Comment:
As I said [here](https://github.com/wordpress-mobile/release-toolkit/pull/443#issuecomment-1402626367), totally agree. I can send a separate PR for cleaning it up as well while I'm at it, I think it would be nice to remove all the references to the deprecated localisation code when we release a new version.